src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public class Circle extends Figure implements GetArea {
```

Comment:
Тобі не потрібно імплементувати інтерфейс якщо ти наслідуєшся від класа який імплементує його
```suggestion
public class Circle extends Figure {
```